packages/main/src/themes/Dialog.css

```diff
@@ -21,6 +21,14 @@
 	max-width: 100vw;
 }
 
+:host([draggable]) .ui5-popup-header-root {
```

Comment:
We should also apply the correct cursor when the user sets a "header" slot. 
Because this CSS applies for the built-in header that shows the "header-text", but doesn't apply to the user defined header via the "header" slot. Explore the ::slotted CSS selector in order to make it.

Reply:
I added it, but the ::slotted selector doesn't allow selecting inner elements deeper in the Shadow DOM, only the most top level inner elements. Because of this, I can't reset the cursor of the inner children like I am doing with the rule
```css
:host([draggable]) .ui5-popup-header-root * {
     cursor: auto
}
```
This results in elements such as headings still inheriting the "move" cursor. A `ui5-button`, for example, will have correct cursor "pointer", because its own CSS defines it.